wg-naming/language-evaluation-framework.md

```diff
@@ -0,0 +1,111 @@
+
+# Language harm evaluation framework
+
+## About
+
+The language evaluation framework is a guidance document developed by the Kubernetes Naming Working Group. It outlines a structured framework for evaluating language and terminology for harm to the community. This enables the community to navigate divisive conversations with a measure of clarity.
+
+While the document was created for an open source technology project, we feel the principles outlines are applicable to other fields as well.
+
+## The framework
+
+### Using the framework
+
+The framework is divided into 3 sections: first-, second-, and third-order concerns, ranked in order of potential for harm the language could cause to the community. 
+
+In general, first-order concerns are egregious, overt, and clearly problematic. Second-order concerns are things which indicate problematic language, but are less clearly provable. Third-order concerns indicate language that could use improvement.
+
+For each term under evaluation, answer all questions.
+
+
+When complete, look at the questions answered in the affirmative: in general, the more questions answered “yes” or “possibly”, the more likely it is that the language in question needs to be erased. If any first-order concerns are a “yes”, replace the language in question. If a significant number of second- or third- order concerns are a “yes”, strongly consider replacing the language in question. 
+
+This framework is intentionally non-prescriptive. Bear in mind that your intention in this work is to reduce harm for the community first and foremost, and let that guide your decisions.
+
+### First-order concerns
+
+First-order concerns are characterized by: 
+
+- Overtness: regardless of its use in the context of code or technology, there is little to no ambiguity outside of technology as to whether the language in question indicates harm.
+- Identity-specificity: language in question specifically unambiguously identifies a group of people.
+
+#### Is the term overtly racist?
+
+Examples include “master/slave”.
+
+#### Is the term overtly sexist, transphobic, or pejorative about a gender identity?
+
+Examples do _not_ include “transclusion” of dependencies, or “binary” operators. 
+
+#### Is the term overtly ableist, or pejorative to neurodiverse or disabled people
+
+Examples include performing “sanity checks”. 
+
+#### Is the term overtly homophobic? 
+
+Examples do not include “homogenizing” or “homogenous” data. 
+
```

Comment:
socioeconomic status is the only category that I see as potentially missing.

Examples include adjectives that label people “the poor” vs specific designations.